htdocs/accountancy/admin/journals_list.php

```diff
@@ -438,7 +438,7 @@
 		print '</tr>';
 
 		// Line to enter new values
-		print '<tr class="oddeven nodrag nodrap nohover">';
```

Comment:
nodrag nodrop

Reply:
Thanks for the hint - I noticed the nodrap case when reviewing the changes (done with codespell ... -w).

I added the minwith100 and minwith200